modules/installation-and-upgrade/pages/container-management/updating-server-containers.adoc

```diff
@@ -1,26 +1,72 @@
 = {productname} Server Upgrade
 
+// FIXME: Shouldn't we better say:
+//        "... to upgrade the host operating system first."
+//        or, maybe:
+//        "... to upgrade the host operating system and especially the
+//         [literal]``mgradm`` tool first."
+
 Before running the upgrade command, it is recommended to upgrade the [literal]``mgradm`` tool first.
 
 .Procedure: Upgrading Server
 . Refresh software repository with [command]``zypper``:
+
 +
+
+[source,shell]
 ----
 zypper ref
 ----
-. Apply available updates with [command]``transactional-update``:
+
+. Depending on the host operating system, proceed with these steps:
+
++
+
+On a transactional system such as {sl-micro} or {leapmicro}: ::
```

Comment:
As this is againts master and we don't have branch yet for 5.1, this could create confusion for MLM users who might think that MLM is also supported on leapmicro